### PR TITLE
Feature/milestone 1

### PR DESCRIPTION
**Pull request by oscardelalanza**
project created with react-create-app, unnecessary files removed, deployed to heroku with mars/create-react-app buildpack

### CODE REVIEW

**Review by fatymahmed — CHANGES_REQUESTED** [review]
## Hi @oscardelalanza, 
You have done a great job in this milestone 👏👏, just a few things to fix for this to be approved
  - Update your readme with relevant information [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)

**Please see my comment below for details**
## Status: Changes Required ♻️
_Feel free to reach out to me on slack (Fatima) if you have any questions or need any help_

File: package.json [hunk 1]
```diff
@@ -6,9 +6,19 @@
...
+    "eslint-config-airbnb": "18.1.x",
+    "eslint-plugin-import": "2.20.x",
+    "eslint-plugin-jsx-a11y": "6.2.x",
+    "eslint-plugin-react": "7.20.x",
+    "eslint-plugin-react-hooks": "2.5.x",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
+    "react-redux": "^7.2.0",
+    "react-scripts": "3.4.1",
```
Comment (fatymahmed) [inline package.json:18]: - `prop-types` should be added to `package.json` see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/section/propTypes.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896806/)

File: README.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
+# Bookstore
+> In this project you will use React and Redux to build a very simple application. You will work in a real-world project
+> for a fictional company that requires your help. The project has been split into multiple milestones, and each milestone
+> has functional and technical requirements that you will need to implement.
```
Comment (fatymahmed) [inline README.md]: - change this part to be about the project.

**Comment by oscardelalanza** [comment]
### Changes
- Prop types added
- Readme updated

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED 

Hi @oscardelalanza good project setup, everything looks great here, you are ready to do great work on this repository :smile:  

**Please merge and move on, This is approved** :+1: 

**RAPHAEL NORIODE** :mask:  stay_safe

**Pull request merged**